scripts/conf.py

```diff
@@ -145,7 +145,7 @@ def nodeset_lines(nodeset, lkp=lkp):
             "State": "UNKNOWN",
             "RealMemory": machine_conf.memory,
             "Boards": machine_conf.boards,
-            "Sockets": machine_conf.sockets,
+            "SocketsPerBoard": machine_conf.sockets,
```

Comment:
We should probably comment in the code that this is reliant on the assumption that Boards=1 is hard coded. Also where it that hardcoded since directly above it is inputing a variable. 

Depending on where the hardcoding is happening (different file or component) then I would be wary of codifying this assumption.

Reply:
I didn't write my description very well and will update. I should have said "Boards is hard-coded to be present". It also happens to be hard-coded to 1. Here is the definition of Boards:

```
Number of Baseboards in nodes with a baseboard controller. Note that when Boards is specified, SocketsPerBoard, CoresPerSocket, and ThreadsPerCore should be specified. The default value is 1.
```

Reply:
New draft handles my concern.